cookbooks/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "vscode-aiconfig.pythonInterpreter": "/opt/homebrew/Caskroom/miniconda/base/envs/aiconfig/bin/python"
+}
```

Comment:
Let's not land these for now since they'll cause merge conflicts

Reply:
Oh, whoops - thanks. Didn't see that sneak in